README.md

```diff
@@ -109,7 +134,7 @@ All versions `0.1.5` onwards require Rails 3.0.x and higher.
 2. Rails 4 compatibility and bye bye Rails 2! Older versions would of course continue to work with Rails 2, but there won't be any support on those.
 
 ## Contributing to `acts_as_list`
- 
+
```

Comment:
Are you able to fix the whitespace differences as they distract from your changes. :) I really appreciate your work on `acts_as_list`! I just want to make sure everything stays tidy and concise :)